kvbc/src/ClientImp.cpp

```diff
@@ -46,7 +46,8 @@ Status ClientImp::start() {
 
 Status ClientImp::stop() {
   // TODO: implement
-  return Status::IllegalOperation("Not implemented");
+  comm_->Stop();
+  return Status::OK();
```

Comment:
I see that ICommunicatio::stop() returns non-zero on failure. Maybe we should accommodate that return code here. What do you think?

Reply:
agreed. updated the PR